### PR TITLE
cc6

### PR DESCRIPTION
**Pull request by ryanapodaca**

### CODE REVIEW

**Comment by ryanapodaca** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ ] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

File: java/datastructures/lib/src/test/java/datastructures/linkedlist/LinkedListTest.java [hunk 1]
```diff
@@ -60,5 +60,56 @@ public class LinkedListTest {
...
+    assertEquals("5", list.toString());
+  }
+
+  @Test
+  public void testAppendMultipleNodes() {
+    LinkedList list = new LinkedList();
+    list.append(5);
+    list.append(10);
+    list.append(15);
+    assertEquals("5 -> 10 -> 15", list.toString());
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/test/java/datastructures/linkedlist/LinkedListTest.java:76]: Make sure all expected results are formatted exactly like the toString method formats its output
